### DISCUSSION TITLE
Winget-Aliases for MS-Store-Packages

### DISCUSSION THREAD

**Discussion by borque** [microsoft/winget-pkgs#90542] [Q&A] [answered]
Hello!
I tried to build packages for Serifs Affinity Publisher (PR #89224 ), Photo (PR #89210 ) and Designer (PR #89223 ). This failed because Serif itself only offers variable download URLs and the available static ones came from external unverified CDNs.

Serif itself says it might provide static download URLs ones they publish MSI-installers (currently they only offer MSIX) and suggests as a temporary solution to use winget to install the above mentioned apps from MS-Store.

winget -search affinity shows they are available from there, but installing them using the cryptic ID is not user friendly. So my question is, whether there can be created alias-names in the format usually used by this winget-manifest-repository simply pointing to the msstore apps or IDs. E.g. serif.affinity.publisher pointing to  and installing 9NTV2DZ11KD9 in/from msstore.

Thanks for help!

**Comment by Trenly** [comment, marked as answer]
At this point in time there isn't a way to create this form of aliases. It is currently being tracked in https://github.com/microsoft/winget-pkgs/issues/29197
